aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -48,27 +47,35 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
         if osutils is None:
             osutils = OSUtils()
 
+        subprocess_npm = SubprocessNpm(osutils)
+        subprocess_esbuild = self._get_esbuild_subprocess(subprocess_npm, scratch_dir, osutils)
```

Comment:
Is there a chance that `subprocess_esbuild` won't be used if the function exits early on L56?

Reply:
Yup, I moved this up intending to change the copy source action to a bundle action since we can now do that without a package.json. Will fix this up.